demisto_sdk/commands/format/update_mapper.py

```diff
@@ -55,3 +61,41 @@ def set_mapping(self):
         """
         if not self.data.get('mapping'):
             self.data['mapping'] = {}
+
+    def extract_content_fields(self, content_fields, built_in_fields):
```

Comment:
demisto_sdk.commands.common.hook_validations.mapper.MapperValidator.is_incident_field_exist
looks like implements very similar logic. In that function, they build a list of invalid_inc_fields_list, and if something is in that list, they return an error. Here, if something is in that list we want to drop it
Lets extract a function there that returns this list. There, we can check if something is in the list and return an error
Here, we can use ` lambda: conatindInList(x) `as our predicate